roles/ovirt-manageiq/README.md

```diff
@@ -0,0 +1,105 @@
+oVirt/RHV ManageIQ/CloudForms deployment
+========================================
+
+This role downloads ManageIQ/CloudForms qcow image and deploys it into oVirt/RHV.
+Create a virtual machine and attach the ManageIQ/CloudForms disk. Then wait for
+initialization of the ManageIQ/CloudForms system and register the RHV/oVirt as
+a infrastructure provider.
+
+Requirements
+------------
+
+oVirt has to be 4.0.4 or higher and [ovirt-imageio](http://www.ovirt.org/develop/release-management/features/storage/image-upload/) must be installed and running.
+The target machine has to have installed [Python SDK](https://pypi.python.org/pypi/ovirt-engine-sdk-python/4.0.4).
+
+Check `ovirt-imageio-proxy` is running on engine:
+
+```
+systemct status ovirt-imageio-proxy
+```
+
+Also check if `ovirt-imageio-daemon` is running on hosts:
+
+```
+systemct status ovirt-imageio-daemon
+```
+
+You also have to use your CA certificate of the engine.
+You must configure the `ovirt_ca` varible with the path to the CA.
+
+Role Variables
+--------------
+
+QCOW varibles:
+```
+ - miq_qcow_url - Define the URL where the ManageIQ/CloudForms qcow is stored (default: http://releases.manageiq.org/manageiq-ovirt-euwe-1.ova).
```

Comment:
Here we have a link to OVA, below you are using link to qcow2, so shouldn't use single term everywhere and just mention in the doc that both OVA or qcow2 formats can be used?